L1Trigger/L1TMuonEndCap/interface/TrackTools.h

```diff
@@ -173,11 +187,18 @@ namespace emtf {
     return loc;
   }
 
-  inline int    calc_phi_loc_int(double glob, int sector, int bits) {  // glob in deg, sector [1-6]
-    assert(bits == 11 || bits == 13);  // 11 for RPCs from CPPF, 13 for CSC LCTs
+  inline int    calc_phi_loc_int(double glob, int sector) {  // glob in deg, sector [1-6]
+    double loc = calc_phi_loc_deg_from_glob(glob, sector);
+    loc = ((loc + 22.) < 0.) ? loc + 360. : loc;
+    loc = (loc + 22.) * 60.;
+    int phi_int = static_cast<int>(std::round(loc));
+    return phi_int;
+  }
+
+  inline int    calc_phi_loc_int_rpc(double glob, int sector) {  // glob in deg, sector [1-6]
     double loc = calc_phi_loc_deg_from_glob(glob, sector);
     loc = ((loc + 22.) < 0.) ? loc + 360. : loc;
-    loc = (loc + 22.) * 15. * pow(2, bits - 11);
+    loc = (loc + 22.) * 60./4.;  // 4x coarser resolution
     int phi_int = static_cast<int>(std::round(loc));
     return phi_int;
```

Comment:
Same here as for RPC theta ... should be phi_int * 4?  How was this working at all before?